crates/ef-testing/src/evm_sequencer/evm_state/v0.rs

```diff
@@ -25,6 +25,11 @@ use crate::evm_sequencer::utils::{
 };
 
 impl Evm for KakarotSequencer {
+    /// Sets up the evm state (coinbase, block number, etc.)
+    fn setup_state(&mut self) -> StateResult<()> {
```

Comment:
what is this for?

Reply:
ah ok! 
you need a specific fn for v1/v0 to setup differently,
we set a coinbase in ssj, whereas in kkt0 we don't,

for now in kkt0, this doesn't do anything

Reply:
can this be done with lazy_static like in #609 at the moment of init?

Reply:
I can rename but I think having something that sets up the "state" (so block, timestamp, coinbase, gas_price,...) makes sense.

Reply:
I can agree that technically setting a coinbase is an "environment" responsibility,
But also, the coinbase "should" to be in the registry, so it's part of the kakarot system

Reply:
not sure I understand, the registry is set during the setup of each individual account, in the `setup_account` function

Reply:
what I could do is: move all storage setting into `setup_state` and only leave in the `lazy_static` what touches the contract classes.